meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -274,6 +274,34 @@ WKS_FILE_apalis-imx8_sota = "sdimage-imx8-sota.wks.in"
 PREFERRED_PROVIDER_u-boot-default-script_apalis-imx8 = "u-boot-base-scr"
 PREFERRED_PROVIDER_u-boot-default-script_sota_apalis-imx8 = "u-boot-ostree-scr-fit"
 
+# iMX8MQ
+UBOOT_SIGN_ENABLE_mx8mq = "1"
+PREFERRED_PROVIDER_virtual/bootloader_sota_mx8mq ?= "u-boot-fio"
+PREFERRED_PROVIDER_u-boot_sota_mx8mq ?= "u-boot-fio"
+PREFERRED_VERSION_u-boot-fio_mx8mq ?= "2020.04"
+PREFERRED_PROVIDER_u-boot-fw-utils_mx8mq = "libubootenv"
+PREFERRED_RPROVIDER_u-boot-fw-utils_mx8mq = "libubootenv"
+SOTA_CLIENT_FEATURES_append_mx8mq = " ubootenv"
+UBOOT_DTB_LOADADDRESS_mx8mq = "0x43000000"
+IMAGE_BOOT_FILES_mx8mq = "imx-boot boot.scr uEnv.txt Image ${@make_dtb_boot_files(d)}"
+IMAGE_BOOT_FILES_sota_mx8mq = "imx-boot boot.itb"
+WKS_FILE_DEPENDS_append_mx8mq = " u-boot-default-script"
+PREFERRED_PROVIDER_u-boot-default-script_mx8mq = "u-boot-base-scr"
+PREFERRED_PROVIDER_u-boot-default-script_sota_mx8mq = "u-boot-ostree-scr-fit"
+KERNEL_IMAGETYPE_sota_mx8mq = "fitImage"
+KERNEL_CLASSES_sota_mx8mq = " kernel-lmp-fitimage "
+OSTREE_KERNEL_ARGS_imx8mqevk ?= "console=tty1 console=ttymxc0,115200 earlycon=ec_imx6q,0x30860000,115200 ${OSTREE_KERNEL_ARGS_COMMON}"
```

Comment:
Changing this to:
OSTREE_KERNEL_ARGS_imx8mqevk ?= "console=tty1 console=ttymxc0,115200 earlycon=ec_imx6q,0x30860000,115200 root=/dev/mmcblk0p2 rootfstype=ext4"